### ISSUE TITLE
Rules: Add support for Able Player transcripts in "Missing Transcript" check

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#1740]
The [Able Player plugin](https://wordpress.org/plugins/ableplayer/) by @joedolson outputs transcripts in specific containers below the video. Currently, videos with Able Player transcripts are flagged as missing transcripts because there is no transcript heading.  We should modify our rule to detect a non-empty Able Player transcript container and not flag missing transcript if it is present.

Here is example code for an empty Able Player transcript container generated by the Able Player WP plugin shortcode:
```
<div id="ableplayer-transcript-able_player_6" class="ableplayer-transcript"><div class="able-transcript-area"><div class="able-window-toolbar able-white-controls"><div class="autoscroll-transcript"><label for="autoscroll-transcript-checkbox-able_player_6">Auto scroll</label><input id="autoscroll-transcript-checkbox-able_player_6" type="checkbox"></div></div><div class="able-transcript"><div class="able-transcript-container" lang="en-US"></div></div></div></div>
```

Example of a non-empty container (this is custom-coded in the theme rather than using the shortcode):
```
<div id="able-player-transcript-4202"><div class="able-transcript-area"><div class="able-window-toolbar able-white-controls"><div class="autoscroll-transcript"><label for="autoscroll-transcript-checkbox-able-player-4202">Auto scroll</label><input id="autoscroll-transcript-checkbox-able-player-4202" type="checkbox"></div><div class="transcript-language-select-wrapper"><label for="transcript-language-select-able-player-4202">Language</label><select id="transcript-language-select-able-player-4202"><option value="en" lang="en">English</option><option value="es" lang="es">Español</option></select></div></div><div class="able-transcript"><div class="able-transcript-container" lang="en"><div class="able-transcript-block"><span class="able-transcript-seekpoint able-transcript-caption" data-start="0.72" data-end="3.365"><span class="able-unspoken">(Announcer)</span> Thank you to our sponsors. </span><span class="able-transcript-seekpoint able-transcript-caption" data-start="3.45" data-end="6.095">Platinum Sponsor: Gravity Forms. </span><span class="able-transcript-seekpoint able-transcript-caption" data-start="6.33" data-end="7.349">Build accessible, </span><span class="able-transcript-seekpoint able-transcript-caption" data-start="7.35" data-end="11.249">powerful WordPress forms
with Gravity Forms-Flexible tools </span><span class="able-transcript-seekpoint able-transcript-caption" data-start="11.25" data-end="15.058">for creators, developers,
and teams of all sizes. </span><span class="able-transcript-seekpoint able-transcript-caption" data-start="15.57" data-end="17.884">Platinum Sponsor: WP Engine. </span><span class="able-transcript-seekpoint able-transcript-caption" data-start="18.12" data-end="20.699">WP Engine provides
the most relied-upon </span><span class="able-transcript-seekpoint able-transcript-caption" data-start="20.7" data-end="23.999">and trusted WordPress
products for businesses, brands, </span>
[...]
</div></div></div></div>
```
This is an example URL with a custom implementation: https://wpaccessibility.day/2025/sessions/accessible-design-patterns-for-2025/

This exception should account for the possibility of multiple Able Player videos on the same page.

**Comment by amberhinds** [comment]
@pattonwebz should there be a linked PR?

**Comment by pattonwebz** [comment]
@amberhinds sorry I closed this by accident. Too many tabs opened and clicked the wrong one to close.

**Comment by pattonwebz** [comment]
@amberhinds a pr is up for this and [has an attached playground link](https://github.com/equalizedigital/accessibility-checker/pull/1741#issuecomment-4674809637) if you want to try it. I added unit tests that match the exact shape of able player transcript markup though so I am pretty confident it solves the issue the customer faced,

**Comment by amberhinds** [comment]
Checked on Playground:
- Video with no transcript added: Flags Missing Transcript ✅
- Video with transcript: Does not flag Missing Transcript ✅
- Video with empty transcript: Flags Missing Transcript ✅

This looks great. Thank you!
